internal/ls/definition.go

```diff
@@ -77,7 +77,7 @@ func (l *LanguageService) ProvideTypeDefinition(
 	}
 	originSelectionRange := l.createLspRangeFromNode(node, file)
 
-	c, done := program.GetTypeCheckerForFile(ctx, file)
+	c, done := program.GetTypeCheckerForFileNonexclusive(ctx, file)
```

Comment:
Ah. I didn't realize this would bubble into the LS code itself. Maybe it was better off the other way around... Whatever you all want, though...